### PR TITLE
Bump submodule | Compile on Ubuntu 22.04

### PR DESCRIPTION
**Pull request by danewalton**
- Bumps submodule to compile with OpenSSL 3.0
- Adds successful gate check for Ubuntu 22.04

### CODE REVIEW

File: build/.vsts-ci.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ variables:
   runCodesignValidationInjection: false
 resources:
   containers:
+  - container: linux-c-ubuntu-2204
```
Comment (ericwol-msft) [inline build/.vsts-ci.yml:6]: blocking until LTS release

**Review by ericwol-msft — CHANGES_REQUESTED** [review]
blocking until LTS release

**Review by ericwol-msft — APPROVED** [review]

**Review by ericwol-msft — APPROVED** [review]

**Review by ewertons — APPROVED** [review]

**Pull request merged**